run_integration_tests.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+IGNORE_HEADER="// INTEGRATION TESTS IGNORE"
```

Comment:
Why not just `// IGNORED`?

Reply:
I wanted to choose something specific enough that for sure will not appear in actual programs... I think it's possible (though highly unlikely) that someone might use `// IGNORED` as an actual comment